app/services/katello/pulp3/content_view_version/metadata_generator.rb

```diff
@@ -17,9 +17,10 @@ def initialize(export_service:)
         def generate!
           ret = { organization: organization.name,
                   repositories: {},
-                  content_view: content_view.slice(:name, :label, :description),
+                  content_view: content_view.slice(:name, :label, :description, :generated_for),
```

Comment:
It's inevitable that we'll be changing the metadata from time to time and  imo we need to make sure that any exports that exist out there today will continue to import without issue - those would be missing generated_for and destination_server per this change

Reply:
Should be fixed!